packages/shared/src/hooks/useReadingHistoryContextMenu.ts

```diff
@@ -0,0 +1,40 @@
+import { useContextMenu } from '@dailydotdev/react-contexify';
```

Comment:
I can see this file as something we can create a generic hook that accepts an id and a generic type to manage the actual data it receives and stored as a state.

We can refactor it now but feel free to just create a ticket to do it in the future as this still provides what we actually need.

Reply:
Thanks Lee for the suggestion. I created a separate task for this. https://dailydotdev.atlassian.net/browse/DD-557